cirq/ops/common_gates.py

```diff
@@ -517,10 +517,6 @@ def _apply_unitary_to_tensor_(self,
     def _decompose_(self, qubits):
         q = qubits[0]
 
-        if self._exponent == 1:
-            yield Y(q)**0.5, X(q)
```

Comment:
Keep this case, but change it to

`cirq.Y(q)**0.5, cirq.XPowGate(global_shift=-0.25).on(q)`